src/api_v2/node.md

```diff
@@ -595,6 +595,109 @@ HTTP/1.1 200 OK
 
       */
 
+== [POST] api/nodes/applyPolicy
+
+    /**
+    @api {post} /api/nodes/{id} 7. Trigger all agent run
+    @apiVersion 8.0.0
+    @apiName applyPolicyAllNodes 
+    @apiGroup Nodes
+
+    @apiParam (Multi) {class}       classes Set class passed as parameter to make it available during agent run.
+
+    @apiDescription This API allows to trigger an agent run on the target node. Response content a json stating if agent has started on each node.
```

Comment:
You need to be more precise on the behavior of the api regarding node answer: what exactly the api is waiting for to build the json answer ?